src/platform-implementation-js/views/section-view.ts

```diff
@@ -37,46 +39,50 @@ function _bindToEventStream(
   sectionViewDriver: GmailCollapsibleSectionView,
   driver: Driver,
 ) {
-  sectionViewDriver.getEventStream().onValue((e) => {
-    sectionView.emit(e.eventName);
+  sectionViewDriver.eventStream.onValue((e) => {
+    if (e != null && 'eventName' in e) {
+      sectionView.emit(e.eventName);
+    }
   });
-  sectionViewDriver
-    .getEventStream()
-    .filter(
-      ({ eventName }: { eventName: unknown }) => eventName === 'rowClicked',
-    )
-    .onValue(
-      ({
-        rowDescriptor,
-      }: {
-        rowDescriptor: { routeID?: string; routeParams: any; onClick: unknown };
-      }) => {
-        if (rowDescriptor.routeID) {
-          driver.goto(rowDescriptor.routeID, rowDescriptor.routeParams);
-        }
-
-        if (typeof rowDescriptor.onClick === 'function') {
-          rowDescriptor.onClick();
-        }
-      },
-    );
-  sectionViewDriver
-    .getEventStream()
-    .filter(({ eventName }) => eventName === 'summaryClicked')
```

Comment:
This looks like it was mapped to `titleLinkClicked` in collapsible-section-view back in d38e019f, but not in this file. We don't depend on it downstream at Streak, but looks like an old bug. 

Otherwise, all event names should match between old and new code.